graphql/api/src/main/resources/stream-registry.graphql

```diff
@@ -33,24 +33,44 @@ type Tag {
 
 ############ Specification ############
 
+input RoleInput {
```

Comment:
Would this be better named `SecurityInput`? Same for `Role` -> `Security`? More consistent with the variable name and that fact it is represents a collection of mappings between roles and principals.

Reply:
OK - renaming done.